fiscalyear.py

```diff
@@ -191,6 +191,16 @@ def __new__(cls, fiscal_year):
         self._fiscal_year = fiscal_year
         return self
 
+    @classmethod
+    def current(cls):
+        """ Alternartive constructor. Returns the current FiscalYear.
```

Comment:
Typo: "Alternartive" -> "Alternative"

Also, can you remove the leading space?